pdns/dnsdistdist/docs/guides/webserver.rst

```diff
@@ -21,6 +21,7 @@ By default, our web server sends some security-related headers::
    X-Permitted-Cross-Domain-Policies: none
    X-XSS-Protection: 1; mode=block
    Content-Security-Policy: default-src 'self'; style-src 'self' 'unsafe-inline'
+   Connection: close
```

Comment:
I don't think `Connection: close` belongs here, it's not a security-related header at all.